### DISCUSSION TITLE
2022 Call for AOSC OS Wallpapers

### DISCUSSION THREAD

**Discussion by MingcongBai** [AOSC-Dev/aosc-os-abbs#4053] [General]
Welcome to the 2022 AOSC OS call for wallpapers!

Please submit your wallpaper in this discussion thread according to the format below:

1. Upload your wallpaper submission in _non-compressed_ form to an image hosting service (such as Imgur), and paste direct link in your response.
2. Note the following under _each_ of your submissions: title of wallpaper, your name and e-mail, and licensing terms.

We stipulate the following rules and expectations for your submission(s):

1. You must be the original author of your submission, though you may submit photographs or graphic designs.
2. Your submission(s) must be of 3000x2000 or higher resolution, unless they are used as tiled patterns.
3. Your submission must not contain trademarks or (virtual or real-life) human(oid) portraits, except for An-An and Tong-Tong.
4. Your submission must not contain political, racial, or religious opinions.

**Note: The rules and expectations above are subject to interpretation by our community contributors. After the submission deadline, community contributors will convene to audit and filter all submissions based on rule compliance and quality.**

Submissions that passed contributor audition will move on to the voting stage, winner of which will become the new default wallpaper for AOSC OS. We will include the rest of the audited submissions in this year's wallpaper collection.

**Submissions will close at 0:00 on July 9th, UTC+8 time, we will publish voting rules and threads in the meantime.**

Thank you again for your submission!

---

欢迎参与 AOSC OS 2022 年度壁纸征集！

请于本讨论帖中投稿，格式如下：

1. 将要投稿的壁纸上传至 **未经压缩** 的图床（如 Imgur）上，并在回复中粘贴图片直链；
2. 在 **每张** 壁纸链接下注明如下信息：壁纸标题、作者名称及邮箱地址，以及壁纸许可；

本次壁纸征集要求及规则如下：

1. 投稿仅限原创作品，但格式不限（摄影作品或平面设计均可）；
2. 除非作为平铺图案使用，壁纸分辨率必须高于 3000x2000 分辨率；
3. 投稿不得包含商标及（虚拟及现实生活中）除安安/同同外的肖像；
4. 投稿不得包含政治、种族、宗教等观点性内容；

**注：如上规则解释权归社区贡献者所有。在征集截止后，社区贡献者将召开例会，届时我们将根据合规性及质量考虑筛选壁纸投稿。**

通过贡献者筛选的壁纸将进入投票环节，得票最高的将成为新的 AOSC OS 默认壁纸，其余所有壁纸将收录至本年度壁纸包中。投票规则将于 UTC +8 时间 7 月 9 日公布。

**投稿截止时间：UTC +8 时间 7 月 9 日零时。**

感谢您的投稿！

**Comment by bill9108** [comment]
https://live.staticflickr.com/65535/52194845420_5e9f43f3e2_o.jpg
Golden Gate | Lin Zhimin | gzlinzhimin@gmail.com | CC BY-NC-ND

https://live.staticflickr.com/65535/52194382263_26f838e7d1_o.jpg
华灯初上 | Lin Zhimin | gzlinzhimin@gmail.com | CC BY-NC-ND

https://live.staticflickr.com/65535/52194382113_2eb0c5e84f_o.jpg
Oh Deer! (Said Jeff) | Lin Zhimin | gzlinzhimin@gmail.com | CC BY-NC-ND

https://live.staticflickr.com/65535/52193353597_0fa3dc5038_o.jpg
Full Moon | Lin Zhimin | gzlinzhimin@gmail.com | CC BY-NC-ND

https://live.staticflickr.com/65535/52194661879_cc3e0a1ac3_o.jpg
拨云见日 | Lin Zhimin | gzlinzhimin@gmail.com | CC BY-NC-ND

**Comment by szclsya** [comment]
https://imgur.com/a/WrNSjD0
Spring | Leo Shen | i@szclsya.me | CC BY-NC-ND

https://imgur.com/a/5gyhDuo
Dusk | Leo Shen | i@szclsya.me | CC BY-NC-ND

**Comment by M0gician** [comment]
https://m0g1cian.piwigo.com/uploads/s/q/5/sq5iv3mjmv//2022/07/05/20220705044031-1eca4b06.jpg
Summer · UMass | m0g1c1an | [m0g1cianyang@gmail.com](mailto:m0g1cianyang@gmail.com) | CC BY-NC

https://m0g1cian.piwigo.com/uploads/s/q/5/sq5iv3mjmv//2022/07/05/20220705043641-a5c575e1.jpg
Lone Wild | m0g1c1an | [m0g1cianyang@gmail.com](mailto:m0g1cianyang@gmail.com) | CC BY-NC

https://m0g1cian.piwigo.com/uploads/s/q/5/sq5iv3mjmv//2021/08/08/20210808075130-9c217812.jpg
三藩剪影 | Jeff Bai | [m0g1cianyang@gmail.com](mailto:m0g1cianyang@gmail.com) | CC BY-NC

https://m0g1cian.piwigo.com/uploads/s/q/5/sq5iv3mjmv//2021/08/09/20210809101417-2596ee34.jpg
St. Brigid's Parish | m0g1c1an | [m0g1cianyang@gmail.com](mailto:m0g1cianyang@gmail.com) | CC BY-NC

https://m0g1cian.piwigo.com/uploads/s/q/5/sq5iv3mjmv//2022/07/05/20220705050841-d7659fd4.jpg
Congregation Mickve Israel | m0g1c1an | [m0g1cianyang@gmail.com](mailto:m0g1cianyang@gmail.com) | CC BY-NC

**Comment by cthbleachbit** [comment]
https://static.cth451.me/DCIM/US/MA/Boston/Land-Boulevard-at-Longfellow-Bridge.jpg
Land Boulevard at Longfellow Bridge | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/NY/Irondequoit-Webster/Irondequoit-Wayne-County-Expressway.jpg
Irondequoit-Wayne County Expressway | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/OH/Cleveland/Tower-City-Center.jpg
Tower City Center | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Devon/Strafford-Station.jpg
Strafford Station | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Devon/Train-to-Malvern.jpg
Train to Malvern | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Erie/Big-Steps.jpg
Big Steps | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Erie/Presque-Isle-Beach-11.jpg
Presque Isle Beach 11 | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Erie/Presque-Isle-Lighthouse.jpg
Presque Isle Light | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Erie/Take-a-Peek.jpg
Take a Peek | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/PA/Philadelphia/South-Vaulted-Walkway.jpg
South Vaulted Walkway | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/VA/Shenandoah/Luray-in-Ray.jpg
Luray in Ray | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/VA/Shenandoah/Luray-in-Twilight.jpg
Luray in Twilight | cth451 | cth451@gmail.com | CC BY-NC

https://static.cth451.me/DCIM/US/VA/Shenandoah/Oat-Grass.jpg
Oat Grass | cth451 | cth451@gmail.com | CC BY-NC

**Comment by rickliu2000** [comment]
https://pastebin.aosc.io/paste/I0-qUPqvL640TxTIUZfUcA
Lake Mendota Blue| Ruikai Liu | rickliu2000@outlook.com | CC BY-NC

https://pastebin.aosc.io/paste/sbsEMfuNbZKrjUyyJp2x7g
Lakeshore Early Spring | Ruikai Liu | rickliu2000@outlook.com | CC BY-NC

https://pastebin.aosc.io/paste/GX~MR9bnDE4f86wh4XGqDA
Capitol Dawn  | Ruikai Liu | rickliu2000@outlook.com | CC BY-NC

https://pastebin.aosc.io/paste/~vf2hShYFfBFA7mWUMTtEg
Lakeshore Bike Path  | Ruikai Liu | rickliu2000@outlook.com | CC BY-NC

**Comment by Nyovelt** [comment]
https://pan.aaaab3n.moe/api/v3/file/source/28/Material%20Tower.jpg?sign=8MKYAEPzLA3bMvGv1RSZLVhWXoKH9sHsQqNQJYxhkSY%3D%3A0
Material Tower | ALANZ | 743387789@qq.com | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/35/Material%20Tower%202.jpg?sign=v5qqxUKFGJBc4GT8I1YNXMiXD29MffFUxqvrtQlCYo0%3D%3A0
Material Tower 2 | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/29/Tux.jpg?sign=QjkM9X6CxMs_AhuPfIm5hDKiTq6B6sX05JWGfcQ1Tpc%3D%3A0
Tux | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/30/Shanghai%20Clearing%20House.jpg?sign=BNiFurMwqipbF0bvBekO7VX72y4kGncPj35wjab_6XY%3D%3A0
Shanghai Clearing House | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/34/SIST.jpg?sign=0rO2gSwH3Ct9aTfX3k6JI8SfxOq0NIqaxU8OP9E1Exo%3D%3A0
SIST | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/37/Moon%20Earth.jpg?sign=lUBAyEbaVVmCJghWRfFUXR1hU-oicpziciuvJXmgBDI%3D%3A0
Moon&Earth | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/38/DNA.jpg?sign=_f956FYB-K9lwLMEGNpfFYIbJtPFpjoilWhW0Vn_XfM%3D%3A0
DNA | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/36/sky-4.jpg?sign=71MxSPq4lerAcnh84PbWAASFOk6m9V-FhP12Lg19zP4%3D%3A0
sky | CZ | captainzst@163.com | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/39/windows.jpg?sign=xS8YmFzmYGGWoJY-insn963Yu7y1dbT3_AA5aOlmOTU%3D%3A0
windows | CZ | captainzst@163.com | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/40/Moon.jpg?sign=lwfCX_OcLCc1tmG15bj9N6eOfMRb7NNi5a1m7wgbPw0%3D%3A0
Moon | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/41/PS.jpg?sign=BjfjW_itR66KRMurOA3Abll9knGnEfgo8e0ARfyEM8U%3D%3A0
Adobe Master Edition | Canarypwn | canarypwn@aosc.io | CC BY-NC

https://pan.aaaab3n.moe/api/v3/file/source/44/NVIDIA%203.jpg?sign=1DwIye6lMTJ7X6UoZTWuEN50T1c8lwrDSCq_TbPLMJI%3D%3A0
NVIDIA | Canarypwn | canarypwn@aosc.io | CC BY-NC

**Comment by neruthes** [comment]
https://oss.neruthes.xyz:2096/p/misc/Rainy_Lily.jpg-395b38f33c9f55e8d96b17aeb61054a5.jpg
Rainy Lily | Neruthes | i@neruthes.xyz | CC BY-NC-ND

https://oss.neruthes.xyz:2096/p/misc/Aqua_Blue.png-435caa18f039c738d5073e455d40db9d.png
Aqua Blue | Neruthes | i@neruthes.xyz | CC BY-NC-ND

**Comment by startmenu** [comment]
https://imgur.com/gallery/niAXxKM
夕阳青草坡•苜蓿 | ZiXu | 909094871@qq.com | CC-BY-NC
https://imgur.com/gallery/teNPT9j
蜂和日丽 | ZiXu | 909094871@qq.com | CC-BY-NC
https://imgur.com/gallery/7AL1dyz
金鸡菊 | ZiXu | 909094871@qq.com | CC-BY-NC
https://imgur.com/gallery/Jlstv5k
蟋戏花间 | ZiXu | 909094871@qq.com | CC-BY-NC

**Comment by MingcongBai** [comment]
https://i.imgur.com/EjYhie3.jpg
Timothy / 猫尾草 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/gbD3gxQ.jpg
Clover Rover / 三叶迷踪 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/2AWQRQG.jpg
Road to Bliss / 乐途 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/ck3fMSC.jpg
Blissful Electrons / 乐与电光 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/ajK3yRY.jpg
Blissful Dusk / 陶陶霞光 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/8mn34BK.jpg
Bliss / 逍遥 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/N0TrwOZ.jpg
Sunset over the Empire / 帝国日暮 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/D8gXs8P.jpg
Forsaken Frontier / 末日放逐 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/TMsJkZ2.jpg
Voxels / 方圆之地 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/lwSN9Uf.jpg
Chicane / 流光回转 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/fyLjKn3.jpg
The World Beyond / 异域视界 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/CaZ5ntb.jpg
Desert Rain / 甘霖 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/eaSLrTy.jpg
Basking in Nature / 光荫 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/F0gpXCN.jpg
Resiliance / 漠林 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/ouO5aaR.jpg
Collapse / 屈服 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/zuBEW2s.jpg
Thorns of Life / 狼棘 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/fMoiG9c.jpg
Spring Chicane / 峰回路转 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/TL5lQL2.jpg
Verdant Sea / 海角逢生 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/QsDQu0B.jpg
Windward Gaze / 神往 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/80gUDcW.jpg
Sentinels of Wind / 风岚 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/nDlPijk.jpg
Closing Time / 落幕 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/WACz0RT.jpg
Night Light / 峦下霓虹 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/y9Rks0q.jpg
To the Cosmos / 问天 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/3YV8x6k.jpg
Regina Caeli / 天擎 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/VA7FlOp.jpg
Neon of the Seas / 鬼屿 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/2MFZROS.jpg
Toxic Ecstacy / 毒艳 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/hJJLO0Y.jpg
Night at the Pub / 酒干倘卖无 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

https://i.imgur.com/PpmRXc2.jpg
Mighty Little People / 人间 | Mingcong Bai | jeffbai@aosc.io | CC-BY-NC

**Comment by megabitsenmzq** [comment]
https://imgur.com/a/o0moBFU
Piece of light | CC-BY-NC

**Comment by luyanaa** [comment]
https://i.imgur.com/sqnYcAZ.jpg
Pudong Airport / 浦东机场 | Robin Lu |  robinluaa@outlook.com | CC-BY-NC

**Comment by RedL0tus** [comment]
https://cavern.v2bv.net/wallpapers-2022/all_is_well.jpg
All is Well / 一切皆好 | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

https://cavern.v2bv.net/wallpapers-2022/barren_solstice.jpg
Barren Solstice / 不知夏 | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

https://cavern.v2bv.net/wallpapers-2022/nightfall.jpg
Nightfall / 夜幕初落 | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

https://cavern.v2bv.net/wallpapers-2022/nixie_114514.png
114514 or any other number / 114514 或其它任何数字 | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

https://cavern.v2bv.net/wallpapers-2022/rainbow_tree.jpg
Rainbow Tree / 彩虹树 | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

https://cavern.v2bv.net/wallpapers-2022/sonne.jpg
Sonne / 赞美太阳（笑） | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

https://cavern.v2bv.net/wallpapers-2022/viridescent_tango.jpg
Viridescent Tango / 绿 | TheSaltedFish | kaymw@aosc.io | CC BY-NC-SA

**Comment by MingcongBai** [comment]
For the record - @MariaBombardier encountered Internet connectivity issues and has thus failed to upload before deadline. Locking this discussion after said submission.

**Comment by MariaBombardier** [comment]
https://imgur.com/a/oHOPHxd
Beach / 沙滩 | MariaBombardier | dev@crh380d.de | CC0
Mai Island / 麦岛 | MariaBombardier | dev@crh380d.de | CC0
KJ-2000 in residential buildings / 在住宅楼间的空警2000 | MariaBombardier | dev@crh380d.de | CC0
Old times in Shanghai Hongqiao Station / 往日浮光 | MariaBombardier | dev@crh380d.de | CC0
A corner of the harbour / 港湾一隅 | MariaBombardier | dev@crh380d.de | CC0
Watching you / 就看着你 | MariaBombardier | dev@crh380d.de | CC0
Yellow Sea Vision / 黄海仙境 | MariaBombardier | dev@crh380d.de | CC0
Mount Lao (Close up) / 崂山近景 | MariaBombardier | dev@crh380d.de | CC0
Bottle-Neck Channel / 宝瓶口 | MariaBombardier | dev@crh380d.de | CC0
Dujiangyan City / 远眺都江堰市 | MariaBombardier | dev@crh380d.de | CC0
Fish Mouth Levee / 鱼嘴 | MariaBombardier | dev@crh380d.de | CC0

**Comment by MingcongBai** [comment]
Thanks, everyone, for your submissions! Locking topic.

**Comment by MingcongBai** [comment]
We have just completed reviewing and auditing your submissions (97 of them, wow!). While we have admitted the vast majority of your submissions, we have rejected some of them for quality and compliance reasons, some submissions might require your revision.

I will now contact you regarding the details. Wallpaper vote will take place next Thursday (July 14th, 2022) on Telegram, please look for update on our chat channels.

For the sake of transparency, you may find the [minutes](https://wiki.aosc.io/developer/minutes/20220709/) on our auditing session today.
